cardinal/testutils/test_utils.go

```diff
@@ -125,7 +125,16 @@ var (
 	privateKey *ecdsa.PrivateKey
 )
 
-func uniqueSignature() *sign.SignedPayload {
+func UniqueSignature(optionalPersonaTagName ...string) *sign.SignedPayload {
```

Comment:
Using variadic parameters for optional parameters is not standard. I'd suggest making a second function:

```
UniqueSignatureWithPersonaTag(personaTag string) *sign.SignedPayload {...}
```